server/modules/slack/slackIntegration.js

```diff
@@ -160,3 +168,27 @@ export const addToChannel = async (fellowEmail, channel) => addOrRemove(fellowEm
  * @returns {Object} - The result after a user has been removed from a channel
  */
 export const removeFromChannel = async (fellowEmail, channel) => addOrRemove(fellowEmail, channel, 'kick');
+
+/**
+ * @function onboardingAutomation
+ * @desc combines all onboarding functions
+ *       into one function.
+ *
+ * @param {string} fellowEmail - user's email
+ * @param {string} channel - the channel id
```

Comment:
I think you need to be more specific about the channel you are referring to here:
```
@param {string} channel - the channel id
```
A name like `removalChannelID` might suffice(just a suggestion).

Reply:
I have added that it is the ID of the channel you are removing the developer for